python/ray/serve/api.py

```diff
@@ -890,6 +894,57 @@ async def __del__(self):
     return decorator
 
 
+@PublicAPI(stability="alpha")
+class DAGHandle:
+    """Resolved from a DeploymentMethodNode at runtime.
```

Comment:
To be accurate i think we should say this is the DAGNode of user chose acting as the root, where all DAGNodes referenced becomes children. It doesn't have to be deployment or deployment method node in particular. We support using a plain ray task as well.